### PR TITLE
Add aws volume name tag check

### PR DESCRIPTION
**Pull request by ebondare**

### CODE REVIEW

**Comment by ebondare** [comment]
Successful test run: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/26321/consoleFull

**Review by yitzhak12 — APPROVED** [review]

File: ocs_ci/ocs/exceptions.py [hunk 1]
```diff
@@ -618,3 +618,7 @@ class TerrafromFileNotFoundException(Exception):
 
 class IncorrectUIOptionRequested(Exception):
     pass
+
+
+class UnexpectedAttributeValue(Exception):
```
Comment (fbalak) [inline ocs_ci/ocs/exceptions.py:623]: It would be better to use Python `ValueError` instead of custom exception.

**Review by fbalak — APPROVED** [review]

File: ocs_ci/ocs/exceptions.py (hunk 1 above)
Comment (ebondare) [reply ocs_ci/ocs/exceptions.py]: Thank you, fixed

**Review by yitzhak12 — APPROVED** [review]

**Comment by ebondare** [comment]
Latest successful test run: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/26351/console

**Pull request merged**